src/main/java/com/MissSherlockHolmes/guessing/game/controller/GameController.java

```diff
@@ -0,0 +1,30 @@
+package com.MissSherlockHolmes.guessing.game.controller;
+
+import com.MissSherlockHolmes.guessing.game.model.Game;
+import com.MissSherlockHolmes.guessing.game.service.GameService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.Map;
+
+@RestController
+public class GameController {
+
+    @Autowired
```

Comment:
Let's not use @Autowired on fileds as it is deprecated method of DI. 

Let's use: 
1. Constructor based injection directly.

`public GameController(GameService gameService) {
   this.gameService = gameService
}`
2. Lombok @RequiredArgsConstructor annotation.